functions-methods.js

```diff
@@ -33,4 +68,31 @@
 // checkEmailValidity("tessmellink@novi.nl") geeft true - want @ en punt op de juiste plek
 // checkEmailValidity("n.eekenanovi.nl") geeft false - want geen @
 // checkEmailValidity("n.eeken@novinl.") geeft false - want de punt mag niet als laatst
-// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
\ No newline at end of file
+// checkEmailValidity("tessmellink@novi,nl") geeft false - want er staat een komma in
+
+const emailAddress = ["n.eeken@novi.nl", "tessmellink@novi.nl", "n.eekenanovi.nl", "n.eeken@novinl.", "tessmellink@novi,nl"];
+
+number = 0
+
+function getEmailDomain() {
+    for (address in emailAddress) {
+
+        if (emailAddress[number].includes("@")) {
+            if (emailAddress[number] != ",") {
+                if (emailAddress[number].charAt(emailAddress[number].length-1) != "."){
+                    console.log("True")
+                }
+            }
+            
+        }
```

Comment:
Dit kun je nog iets versimpelen
```suggestion
        console.log(address.includes("@") && address[address.length - 1] !== "," && address[address.length - 1] !== ".");
```